src/main/web_src/src/components/ui/etikett/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import cn from 'classnames'
+import Etikett, { EtikettBaseProps } from 'nav-frontend-etiketter'
+
+import './etikett.less'
+
+interface ImportFraEtikett {
+	importFra: string
+	type: EtikettBaseProps
+	venstreMargin?: boolean
+}
+
+const importType = (fra: string) => {
+	switch (fra) {
+		case 'TPS':
+			return 'Mini-Norge'
+		default:
+			'Ukjent'
+	}
+}
+
+const ImportFraEtikett = ({ importFra, type, venstreMargin }: ImportFraEtikett) => {
+	if (!importFra) return null
+
+	const css = cn('dolly-etikett', { 'dolly-etikett--venstre-margin': venstreMargin })
+	return (
+		<Etikett mini className={css} type={type} title={`Importert fra ${importType(importFra)}`}>
```

Comment:
Får warning "Received `true` for a non-boolean attribute `mini`", men skjønner ikke helt hvorfor, for den skal jo være boolean?

Reply:
Aha! Nå ser ihvertfall etikettene helt nydelige ut, og feilmeldingen er borte 😄